### ISSUE TITLE
a way to make font (and icons/buttons?) bigger

### ISSUE THREAD

**Issue by goffi-contrib** [snikket-im/snikket-ios#201]
For accessibility reason, it should be possible to make fonts and icons/buttons bigger: some users of my instance have vision troubles, and they can't read Snikket font correctly.

This feature is an important accessibility need: the app is unusable for some users because of this.

There is a related ticket at https://github.com/snikket-im/snikket-ios/issues/184. I'm not familiar with iOS myself, thus I'm not sure if it's better to use per-app size or to have directly in parameter a simple way to make fonts and icons bigger. If per-app size is better, this ticket can be closed in favour of #184 . Anyway a simple way to make everything bigger should be available and easy to set, even of older phones.

Thanks
